scripts/create-relocatable-package.py

```diff
@@ -45,9 +45,9 @@ def reloc_add(self, name, arcname=None, recursive=True, *, filter=None):
 output = args.dest
 
 ar = tarfile.open(output, mode='w|gz')
-# relocatable package format version = 2
+# relocatable package format version = 2.2
 with open('build/.relocatable_package_version', 'w') as f:
-    f.write('2\n')
+    f.write('2.2\n')
```

Comment:
Will this require changes elsewhere? Or is it just infomative?

Reply:
we will use that in CCM, the whole change is aimed for CCM... (where those systemd files are "escaping" into the user directory, and can cause conflicts in some cases)

Reply:
Is ccm already prepared?

Reply:
for this change, no.

we'll do the changes in CCM only after those ones are merge. i.e. it's not a breaking change. (shouldn't be... an scylla CI is covering it, i.e. running dtest)